pkg/kubelet/server/auth_test.go

```diff
@@ -128,6 +128,7 @@ func AuthzTestCases() []AuthzTestCase {
 		"/metrics/cadvisor":                   "metrics",
 		"/metrics/probes":                     "metrics",
 		"/metrics/resource/v1alpha1":          "metrics",
+		"/metrics/resource":                   "metrics",
```

Comment:
I haven't look at this test case carefully, I can't see any benefit, I will be appreciated if anyone who can tell me.